packages-exp/auth-exp/src/core/auth/auth_impl.ts

```diff
@@ -268,6 +275,14 @@ export class AuthImpl implements Auth, _FirebaseService {
     return this.assertedPersistence.persistence.type;
   }
 
+  _updateErrorMap(errorMap: AuthErrorMap): void {
```

Comment:
since the dep is required at initialization, can we make this immutable and in the constructor?

Reply:
No; the instance is actually constructed here: https://github.com/firebase/firebase-js-sdk/blob/master/packages-exp/auth-exp/src/core/auth/register.ts#L67

We don't have access to any of the specific auth deps at that point.